doc/tutorials/duckdb-native-sqlalchemy.md

```diff
@@ -0,0 +1,134 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.7
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: JupySQL and DuckDB with SQLAlchemy vs native connection
+    keywords: jupyter, jupysql, duckdb, sqlalchemy
+    property=og:locale: en_US
+---
+
+# DuckDB (native vs SQLAlchemy)
+
+> tl;dr; If using DuckDB, use a native connection. Only use SQLAlchemy for legacy projects
```

Comment:
`tl;dr;` can be removed

Reply:
the tl;dr expression is pretty common and it helps to tell the reader that the message is a summary, in case they want to skip reading it

Reply:
reformatted as an admonition